docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -13,154 +13,220 @@ isTocNested: true
+Note from the above cluster configuration that the nodes in us-east-1a were 172.152.29.181, 172.152.27.126 and 172.152.22.180. By navigating to the table view in the YB-Master UI, we can verify that the tablet created for this table was indeed placed in us_east_1a_zone:
 
-### How to Create a Single-Zone Tablespace
+![YB-Master UI: Tablets of single_zone_table](/images/explore/tablespaces/single_zone_table_tablet_distribution.png)
 
-To highlight the difference between multi-zone and single-zone tablespaces, the following example shows how to create a single-zone tablespace:
+Now let us measure the latencies incurred for INSERTs and SELECTs on this table, where the client is in us-east-1a zone:
 
 ```sql
-CREATE TABLESPACE us_west_1a_tablespace 
-WITH (replica_placement='{"num_replicas": 3, "placement_blocks":
-[{"cloud":"aws","region":"us-west","zone":"us-west-1a","min_num_replicas":3}]}');
+yugabyte=# INSERT INTO single_zone_table VALUES (1, 'field1'), (2, 'field2'), (3, 'field3');
 ```
 
-If you create a table that uses `us_west_1a_tablespace`, each tablet of the table will have three replicas, but unlike multi-zone tablespaces, all three replicas for any tables and indexes in this tablespaces will be placed within the `us-west-1a` zone.
+```output
+Time: 4.676 ms
+```
 
-## Creating Tables and Indexes in Tablespaces
+```sql
+yugabyte=# SELECT * FROM single_zone_table;
+```
 
-You can associate new tables and indexes with a corresponding tablespace. This defines the replication factor of the table or index. It also defines how the replicas of the table or index are to be spread across cloud, regions, and zones.
+```output
+ id | field
+----+--------
+  2 | field2
+  1 | field1
+  3 | field3
+(3 rows)
 
-A table and an index can be created in separate tablespaces.
+Time: 1.880 ms
+```
 
-### How to Use a Multi-Zone Tablespace
+## Create a multi-zone table
 
-Using the multi-zone tablespaces defined in [How to Create a Multi-Zone Tablespace](how-to-create-a-multi-zone-tablespace), you can apply the YSQL's `TABLESPACE` option to `CREATE TABLE` and `CREATE INDEX` statements, as follows:
+Now let us create a table that is spread across multiple zones within the same region as shown in the following diagram:
```

Comment:
The following diagram is a graphical representation of a table that is spread across multiple zones within the same region: